router.go

```diff
@@ -164,6 +164,7 @@ func (s *PhilRouter) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	w.Header().Set("Access-Control-Allow-Methods", "POST, GET, OPTIONS, PUT, DELETE")
 	w.Header().Set("Access-Control-Allow-Headers", "Accept, Content-Type, Content-Length, Accept-Encoding, X-CSRF-Token, Authorization, X-Requested-With, X-App-Source, X-Request-Id, X-User-Id, Strict-Transport-Security, X-Forwarded-For, X-Real-Ip, Browser-User-Agent")
 	w.Header().Set("Access-Control-Max-Age", "7200")
+	w.Header().Set("Content-Security-Policy", "default-src 'self'; script-src 'self'; object-src 'self';style-src 'self' img-src 'self'; media-src 'self'")
```

Comment:
need help with one. Do we have any list of allowed sources ?

Reply:
@basar-phil are we talking about 
```
http://localhost:9292, http://localhost:3000, http://localhost:8080, https://int.dev.phil.us/, https://my.dev.phil.us, https://ops.dev.phil.us, https://web.dev.phil.us, https://md.dev.phil.us, https://us.dev.phil.us,https://workflow.dev.phil.us, https://data.dev.phil.us, https://gifted-goldberg-e5cca9.netlify.app
```

If so, it is located in application.****.properties with key = cors.allowed.list

Reply:
keeping it default, as per @hammad

Reply:
@basar-phil using self for everything might not work, since we use resources from other domains too. eg, for fonts, we use fonts.googleapis.com or fonts.gstatic.com. we also use other third party tools like fullstory and mixpanel, which have js files loading from different domains.

ideally, you'd want to install tools like https://chrome.google.com/webstore/detail/content-security-policy-c/ahlnecfloencbkpfnpljbojmjkfgnmdc and browse our frontend websites and let it gather all the domains that the site requires and generate the policy.

or as a last resort, we can also have some of the sources as *, eg, `img-src *`.